data/installation-data-1.1.yml

```diff
@@ -302,6 +302,7 @@
   download_method: Package manager
   architecture: universal
   installation_code: npm install duckdb@next
+  note: The nightly release of the Node.js driver installs the old Node.js driver and not DuckDB Node Neo. For the Node Neo driver, the nightly release is currently not available. If you require this driver, consider compiling it from source.
```

Comment:
Minor comment: Compiling the Node Neo driver from source won't get you a nightly version, because the build depends on released DuckDB binaries, and currently doesn't pull in nightly binaries.

It probably wouldn't be hard to add an option to pull in nightly binaries, but that configuration doesn't exist today.

Reply:
Thanks for the clarification! I'll just remove this part of the note then.